scripts/build-ios-arm64.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# Copyright 2019 Google LLC
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -e
+
+if [ ! -d "/Library/Developer/CommandLineTools" ] || [ ! -x "$(command -v xcodebuild)" ] 
+then
+  echo "Have you installed Xcode?"
+  exit 1
+fi
+
+mkdir -p build/iOS/arm64
+
+# Create Tool-chain file
+IOS_TOOL_CHAIN=build/iOS/arm64/ios_tmp.toolchain.cmake
+cat << EOF > ${IOS_TOOL_CHAIN}
+set(CMAKE_SYSTEM_NAME iOS)
```

Comment:
If you need a toolchain file, create on in the top-level `cmake` subdirectory.

Reply:
removed.